native/src/modules/endpoint/reducers/pushCategory.js

```diff
@@ -46,6 +46,12 @@ const pushCategory = (state: CityContentStateType, action: PushCategoryActionTyp
     }
   }
 
+  // Check whether another page in the same city is loading, e.g. because it is being refreshed.
```

Comment:
Maybe (really just maybe) it would be easier to include whether another page is loading in the action as param? But not sure if possible at all or better. This solution is perfectly fine.